CONTRIBUTING.md

```diff
@@ -0,0 +1,56 @@
+# Contribution Guidelines
+
+Thanks you for considering a contribution to Blendid.
+
+## Getting Started
+
+Before you begin developing on Blendid, you'll need to setup your environment:
+
+1. Install [NodeJS](https://nodejs.org) 6.11.1 or higher.
+2. Install [Yarn](https://yarnpkg.com)
+
+All commands should be run using yarn. If you haven't switched to [yarn](https://yarnpkg.com/en/) yet, now's a great time!
+
+> If you are familiar with npm then using yarn should be a breeze. You can keep using npm if you'd prefer but you will miss out on the safety and security of yarn
+
+Install project dependencies with:
+
+```
+yarn install
+```
+
+That's it! You're all set!
+
+## Deployment
+
+Blendid is distributed via [npm](https://npmjs.com). npm is a package manager for JavaScript.
+
+### Login to the Viget npm account
+
+First, log in to npm using the [Viget](https://www.npmjs.com/~viget) account:
+
+```
+npm login
+username: viget
+password: <secret>
+```
+
+### Publishing to npm
+
+1. Update CHANGELOG.md
+2. npm version patch
+3. npm publish
+4. git push origin --tags 
+5. Add changelog to associated version tag at https://github.com/vigetlabs/blendid/releases
+
+### What did I just do
```

Comment:
`####` to be a subsection of `### Publishing to npm`?

Reply:
good catch. i should probably change the wording on this as well to something like `what these steps do` because it took me a good 3 minutes to figure out what this section is trying to convey